packages/tasit-action/src/TasitAction.js

```diff
@@ -98,6 +107,7 @@ class TransactionSubscription extends Subscription {
   #provider;
   #tx;
   #txConfirmed = false;
+  #timeout = config.events.timeout;
```

Comment:
I can use destructuring always, I've done that way because it's shorter.
If it's better (use destructuring always), I'll change that.
Another question: Should I assign class properties alway from constructor() either?

Reply:
https://github.com/tasitlabs/TasitSDK/pull/136